doc/CamOverlayAPI.md

```diff
@@ -1,10 +1,10 @@
 # CamOverlayAPI
 
-Module for easy control of CamOverlay drawing API. For more details on supported drawing functions see https://camstreamer.com/camoverlay-api1
+Module for access to CamOverlay HTTP interface.
 
 ## Methods
 
--   **CamOverlayAPI(options)** - Options parameter contains access to the camera, service name, service ID and camera. If service ID is not specified, service is automatically created/selected based on serviceName. Specify video channel using parameter camera (in which View Area overlay will be shown). If you need to specify multiple video channels, you can use an array: `camera=[0,1]`. If omitted the default value `camera=0` is used.
+-   **CamOverlayAPI(options)** - Options parameter contains access to the camera and service ID. Values mentioned in example below are default.
```

Comment:
Autorizace je defaultne prazdna, ale nevim jeslti ma cenu nejak ten popis upravit.

Reply:
Rozhodně to má cenu, vzhledem k tomu, že je ten popis neaktuální.
!